test/Serilog.Sinks.PeriodicBatching.Tests/Sinks/PeriodicBatching/PortableTimerTests.cs

```diff
@@ -0,0 +1,59 @@
+﻿#if NETCOREAPP1_0
```

Comment:
`#if !WAITABLE_TIMER` will keep this aligned with the way `PortableTimer` is conditionally included in the project being tested.